Source/MaestroDt.cpp

```diff
@@ -111,37 +121,42 @@ Maestro::EstDt ()
             // lo/hi coordinates (including ghost cells), and/or the # of components
             // We will also pass "validBox", which specifies the "valid" region.
             if (spherical == 0) {
-                estdt(&lev,&dt_grid,&umax_grid,
-                      ARLIM_3D(tileBox.loVect()), ARLIM_3D(tileBox.hiVect()),
-                      ZFILL(dx),
-                      BL_TO_FORTRAN_FAB(sold_mf[mfi]),
-                      BL_TO_FORTRAN_FAB(uold_mf[mfi]),
-                      BL_TO_FORTRAN_FAB(vel_force_mf[mfi]),
-                      BL_TO_FORTRAN_3D(S_cc_old_mf[mfi]),
-                      BL_TO_FORTRAN_3D(dSdt_mf[mfi]),
+#pragma gpu box(tileBox)
+                estdt(AMREX_INT_ANYD(tileBox.loVect()), AMREX_INT_ANYD(tileBox.hiVect()),
```

Comment:
The reduction for both this and firstdt will not work correctly. AMREX_MFITER_REDUCE_MIN makes a copy of the host data for each tile and stores them in a vector over tiles. Then at the end of the MFIter loop, we reduce over that vector. However this code does the reduction on dt_lev immediately after launching the kernel, at which point dt_grid will not have meaningful data (since the GPU work is asynchronous). This code needs to be refactored so that the update to dt_lev happens outside the MFIter loop.